scripts/variantstore/wdl/GvsWarpTasks.wdl

```diff
@@ -53,10 +54,10 @@ task SNPsVariantRecalibratorCreateModel {
         --maximum-training-variants ~{maximum_training_variants} \
         --output-model ~{model_report_filename} \
         --max-gaussians ~{max_gaussians} \
-        -resource:hapmap,known=false,training=true,truth=true,prior=15 ~{hapmap_resource_vcf} \
-        -resource:omni,known=false,training=true,truth=true,prior=12 ~{omni_resource_vcf} \
-        -resource:1000G,known=false,training=true,truth=false,prior=10 ~{one_thousand_genomes_resource_vcf} \
-        -resource:dbsnp,known=true,training=false,truth=false,prior=7 ~{dbsnp_resource_vcf}
+        -resource:hapmap,known=false,training=true,truth=true,prior=0 ~{hapmap_resource_vcf} \
+        -resource:omni,known=false,training=true,truth=true,prior=0 ~{omni_resource_vcf} \
+        -resource:1000G,known=false,training=true,truth=false,prior=0 ~{one_thousand_genomes_resource_vcf} \
+        -resource:dbsnp,known=true,training=false,truth=false,prior=0 ~{dbsnp_resource_vcf}
```

Comment:
naive question: what do these priors mean here?

Reply:
That's temporary - need to do this to use P&S best. I will make a comment - or ideally parameterize this change.